guide/14-deep-learning/object-detection.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>in the first line of the paragraph it would be informative if the word VMTI (<em>output VMTI predictions</em> ) is elaborated  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>